frontend/src/lib/k8s/pod.ts

```diff
@@ -51,7 +51,7 @@ export interface ExecOptions extends StreamArgs {
 
 export interface LogOptions {
   /** The number of lines to display from the end side of the log */
-  tailLines?: number;
+  tailLines?: number | 'all';
```

Comment:
I'd prefer to keep it as is and just use a negative number to indicate "all". Yes, it'll need extra comments explaining the special meaning of negative numbers, but I feel like it's safer in the long run to have just one type being treated.